bin/xbps-remove/main.c

```diff
@@ -93,10 +95,12 @@ state_cb_rm(const struct xbps_state_cb_data *xscd, void *cbdata UNUSED)
 		}
 		break;
 	case XBPS_STATE_SHOW_REMOVE_MSG:
-                printf("%s: pre-remove message:\n", xscd->arg);
-		printf("========================================================================\n");
-		printf("%s", xscd->desc);
-		printf("========================================================================\n");
+		chars_print = printf("=== %s: remove message ", xscd->arg);
+		if (chars_print < sizeof bar) {
+			printf("%s", bar + chars_print);
+		}
+		printf("\n%s", xscd->desc);
```

Comment:
```
		if (chars_print < sizeof bar) {
			puts(bar + chars_print);
		}
                fputs(xscd->desc, stdout); // not sure if you like this one, otherwise keep the printf
```